src/mlpack/tests/local_coordinate_coding_test.cpp

```diff
@@ -10,21 +10,19 @@
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
 
+// Is the comment below still relevant ?
```

Comment:
Yes, but in this case we don't use the plain REQUIRE function but `Approx(0.0).margi(tol)`, so we should update the commend accordingly.

Reply:
Yes will update the comment. I am glad i put up a comment there, I knew i would forget to ask with so many migrations

Reply:
also I am surprised you noticed the comment :o

Reply:
Hey @zoq, I don't think this comment is relevant anymore because `BOOST_REQUIRE_CLOSE` is used in that test [here](https://github.com/mlpack/mlpack/pull/2678/files/88d7f55d7e467d94ea2695e80f2f67c96b45f9b4#diff-5e06511318dffc22fb9856facd2b9fb994c921b2a53afeb92b5c2e684156c3e6L155) and below that. and so i don't think we even require a Catch2 equivalent of that comment.

Reply:
And if that is the case I think this PR is ready for a final review.

Reply:
Okay, will look over the PR.

Reply:
Sure, let me know what you think then.